graphics/opengl/framebuffer.cpp

```diff
@@ -158,12 +158,20 @@ void FrameBuffer::init() {
 		glBindRenderbuffer(GL_RENDERBUFFER, 0);
 	} else {
 		glBindRenderbuffer(GL_RENDERBUFFER, _renderBuffers[0]);
+		#if defined(EMSCRIPTEN)
```

Comment:
I really don't like ifdefs like this. maybe some function which return GL type

Reply:
I changed this and replaced it with a check against GL_VERSION at runtime